index.php

```diff
@@ -53,7 +53,7 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 
-	<title>WP Update Server Stats - <?php echo htmlentities($report->getSlug()); ?></title>
+	<title>WP / CP Update Server Stats - <?php echo htmlentities($report->getSlug()); ?></title>
```

Comment:
I recommend changing this to just "Update Server Stats", without the WP/CP prefix. If there's one WP fork then there could be more in the future and listing every supported fork or CMS in the title would just lead to clutter.